app/code/Magento/Review/Controller/Adminhtml/Product/Save.php

```diff
@@ -73,6 +73,10 @@ public function execute()
             } else {
                 $resultRedirect->setPath('*/*/');
             }
+            $productId = $this->getRequest()->getParam('productId');
```

Comment:
Please use type casting to `(int)` here to limit cases of passing unpredictable values to the URL.

Reply:
Hello @rogyar 
I have updated code as per suggestion